test_proxy/handlers/client_handler_legacy.py

```diff
@@ -79,54 +79,157 @@ async def ReadRows(self, request, **kwargs):
             row_list.append(dict_val)
         return row_list
 
+    @client_handler.error_safe
```

Comment:
Yes, the conformance tests run through the test proxy. This is filling in some gaps in the test proxy implementation for the legacy client

Reply:
Sounds good. Changes here are pretty low risk since they don't affect client behavior so I guess they can be worked on over time until all test cases are handled.